api/nakadi-event-bus-api.yaml

```diff
@@ -392,6 +394,46 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /event-types/{name}/effective_schema:
+    get:
+      tags:
+        - schema-registry-api
+      security:
+        - oauth2: ['uid']
+      description: |
+        When publishing events whose `EventType` category is either "business" or "data", additional fields are required
+        other than those described in the schema.
+
+        This endpoint exposes this effective schema, e.g. the schema used to validate events in fact.
+      parameters:
+        - name: name
+          in: path
+          description: Name of the EventType to update.
+          type: string
+          required: true
+        - name: X-Flow-Id
+          in: header
+          description: |
+            The flow id of the request, which is written into the logs and passed to called services. Helpful
+            for operational troubleshooting and log analysis.
+          type: string
+          format: flow-id
+      responses:
+        '200':
+          description: Ok
```

Comment:
I mean that there is not schema provided and no description of returned object type.